### PR TITLE
Added UX Tests for column / Edit cells / Common transforms

### LINKED ISSUE #3553: Add UX Test for column / Edit cells / Common transforms
This issue is following the work implemented for issue #3423 in [PR 3541](https://github.com/OpenRefine/OpenRefine/pull/3541)

The goal is to cover all functionalities under column / edit cells / common transforms

Screen Shot 2021-02-02 at 09 17 51

### Proposed solution

Write the remaining tests for

- Convert Smart quotes with ascii
- to titlecase
- to uppercase
- to lowercase
- to text
- to null
- to empty string

### PR DESCRIPTION
**Pull request by S-Harshit**
Fixes #3553

Added Tests for:

- Convert Smart quotes with ascii
- to titlecase
- to uppercase
- to lowercase
- to text
- to null
- to empty string

### CODE REVIEW

File: main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+describe(__filename, function () {
+  it('Ensure text is converted to Uppercase', function () {
+    cy.loadAndVisitProject('food.mini');
+
+    cy.columnActionClick('Shrt_Desc', [
```
Comment (kushthedude) [inline main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js:5]: Can we first convert the column to the number and then test the conversion to text?

**Review by kushthedude — CHANGES_REQUESTED** [review]

File: main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+describe(__filename, function () {
+  it('Ensure text is converted to Uppercase', function () {
```
[the remark this answers is not in the document]
Comment (S-Harshit) [reply main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js]: I have changed spec-name.

File: main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js (hunk 1 above)
Comment (S-Harshit) [reply main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js]: I have changed the code as suggested.

File: main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+describe(__filename, function () {
+  it('Ensure number is converted to text', function () {
+    cy.loadAndVisitProject('food.mini');
+
+    cy.columnActionClick('NDB_No', [
```
Comment (kushthedude) [inline main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js:5]: Can we use `cy.castColumnTo` here as defined in https://github.com/OpenRefine/OpenRefine/blob/b5192327495aaee18c64a8a0eb8f70aae5adadb7/main/tests/cypress/cypress/support/commands.js#L101 ? Else the tests looks good to me!

Comment (S-Harshit) [reply main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js]: I have removed cy.columnActionClick() and used cy.castColumnTo() to cast the column to number before testing to-text using cy.columnActionClick().

**Review by kushthedude — APPROVED** [review]
Looks good to me!

**Review by wetneb — APPROVED** [review]
Thank you very much!

**Pull request merged**